translations/messages.fr.yml

```diff
@@ -940,6 +940,7 @@ political_committee.membership.quality.elected_candidate_adherent: Adhérent(e)
 political_committee.membership.adherent_has_already: L'adhérent ({{ email }}) est déjà un membre du Comité politique "{{ politicalCommittee }}"
 political_committee.membership.adherent_has_no_membership: L'adhérent ({{ email }}) en'est pas un membre du Comité politique du Conseil territorial "{{ territorialCouncil }}"
 political_committee.membership.has_max_number_of_mayor_and_leader: Vous ne pouvez nommer que {{ max }} maire ou chef de file dans le Comité politique "{{ politicalCommittee }}".
+political_committee.members_count: '1 membre|%count% membres'
```

Comment:
we already have this value `'1 membre|%count% membres'` in the yaml, maybe we can use it (and renaming for using a generic key)?